src/pages/Projects.tsx

```diff
@@ -7,6 +7,16 @@ import Fuse from "fuse.js";
 import SpinnerLoader from "../components/SpinnerLoader";
 import { IconContext } from "react-icons";
 
+// function to randomly shuffle an array
```

Comment:
Please put this function in a file in `utils/`

Reply:
Done